graphene_django_extras/fields.py

```diff
@@ -13,6 +15,29 @@
 from .paginations.pagination import BaseDjangoGraphqlPagination
 from .utils import get_extra_filters, queryset_factory, get_related_fields, find_field, get_kwarg_processors
 
+DECIMAL_FIELD_NAMES = {'DecimalField', 'DecimalInField'}
+ID_FIELD_PATTERN = r'(^id$|^id_|.*_id_.*|.*_id$)'
+
+def get_filtering_args_from_filterset(filterset_class, type):

```

Comment:
Seems like we should be extending the old functionality instead of rewriting it. Something like:

```
from graphene_django.filter.utils import get_filtering_args_from_filterset as _get_filtering_args_from_filterset

...

def get_filtering_args_from_filterset(filterset_class, type):
  args = _get_filtering_args_from_filterset(filteset_class, type)
  ...new stuff here...
  return args
```